src/main/resources/templates/pebt/confirmBlankEmail.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html th:lang="${#locale.language}">
+<head th:replace="~{fragments/head :: head(title=#{confirm-blank-email.title})}"></head>
+<body>
+<div class="page-wrapper">
+    <div th:replace="~{fragments/toolbar :: toolbar}"></div>
+    <section class="slab">
+        <div class="grid">
+            <div th:replace="~{fragments/goBack :: goBackLink}"></div>
+            <th:block th:if="${true}">
+                <main
+
+                        id="content" role="main" class="form-card spacing-above-35">
```

Comment:
weird spacing/indenting throughout? you may want to check on that.

Reply:
Yeah! I'm using IntelliJ Cmd-Opt-L to reformat files. I like to rely on an autoformatter. I'll see if there are configuration settings I can tweak to make that work nicely for this project; if not, I can think about not using it.

Reply:
<img width="518" alt="image" src="https://github.com/codeforamerica/homeschool-pebt/assets/67708639/fb554119-8907-46eb-acce-41c58dabbf50">

This seems better.